suite-native/storage/src/migrations/walletSettings/v2.ts

```diff
@@ -0,0 +1,33 @@
+import { PersistedState, getStoredState } from 'redux-persist';
+
+import { WalletSettings } from '@suite-common/wallet-types';
+
+import { isPersistedState } from '../../migrationTypes';
+import { initMmkvStorage } from '../../storage';
+
+type MigratedState = Partial<WalletSettings> & PersistedState;
+
+/**
+ * Migrates device.isDeviceAutoEjectEnabled to walletSettings.autoEject
```

Comment:
To prevent confusion..
```suggestion
 * Migrates device.isDeviceAutoEjectEnabled to walletSettings.isAutoEjectEnabled
```

Reply:
fixed by removing useless comment https://github.com/trezor/trezor-suite/pull/22969/commit 🙉

Reply:
Yeah, true!